tests/unit/test_solvers.py

```diff
@@ -1048,15 +1050,238 @@ def solver_factory(self) -> Union[type[Solver], jtu.Partial]:
             kernel=kernel,
         )
 
+    @override
+    @pytest.mark.parametrize("use_cached_state", (False,))
+    def test_reduce(
+        self,
+        jit_variant: Callable[[Callable], Callable],
+        reduce_problem: _ReduceProblem,
+        use_cached_state: bool,
+    ) -> None:
+        """
+        Check `reduce` raises no errors.
```

Comment:
If this refers to the `reduce` method of a class, it should be referenced here, for eg: `coreax.solvers.RPCholesky.reduce`